app/import_data_v2.py

```diff
@@ -10,39 +10,42 @@
 '''
 
 
+def insert_meta(input, type, company_id):
+    if input is not None:
+        # Iterate over the meta type
+        for item in input:
+
+            # Check if the meta string is already in the Meta table
+            item_check = Meta.query.filter_by(
```

Comment:
Another `findOrCreate` right?

Reply:
Again, great idea! Should be added soon to this PR.